src/Main.elm

```diff
@@ -112,15 +114,10 @@ stringFromViewMode viewMode =
             "CollectData"
 
 
-type alias System =
-    { name : String, description : String }
```

Comment:
- don't need `name` here since we're providing `name : Maybe String` directly in `FormField`
- `description` here is a mistake; just use `description : String` directly in `FormField`